CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ x.x.x Release notes (yyyy-MM-dd)
 ### Bugfixes
 
 * Use of KVC collection operators on Swift collection types no longer throws an exception.
+* The contents of `List` and `Optional` properties are now correctly preserved when copying
```

Comment:
They're not preserved when doing anything that results in the objects graph being recursively traversed from Objective-C. I'm not sure what the complete list of operations that do that is, or whether it's beneficial to list them all here.